bitnami/logstash/values.yaml

```diff
@@ -267,10 +267,10 @@ livenessProbe:
     path: /
     port: monitoring
   initialDelaySeconds: 60
-  periodSeconds:
-  timeoutSeconds:
-  successThreshold:
-  failureThreshold:
+  periodSeconds: ""
+  timeoutSeconds: ""
+  successThreshold: ""
+  failureThreshold: ""
```

Comment:
This breaks the installation of logstash: https://github.com/bitnami/charts/pull/7045/checks?check_run_id=3145250534#step:6:1435

Reply:
We usually set some default values for the probes, you can check other charts and apply the most used values

Reply:
Done in c5154d5 (I committed there a debug file by mistake, removed in 535e6e3). I've set the values to those that were suggested in comments before the changes to adapt the file to readme-generator:
https://github.com/bitnami/charts/blob/d6ba56d2fbc9e04728e7f39434213f5814ecf872/bitnami/logstash/values.yaml#L259-L262